python/ql/lib/semmle/python/frameworks/Flask.qll

```diff
@@ -171,6 +171,24 @@ module Flask {
       override DataFlow::Node getMimetypeOrContentTypeArg() { none() }
     }
 
+    /**
+     * A call to `flask.jsonify` function. This creates a JSON response.
+     *
+     * See
+     * - https://flask.palletsprojects.com/en/2.2.x/api/#flask.json.jsonify
+     */
+    private class FlaskJsonifyCall extends InstanceSource, DataFlow::CallCfgNode {
+      FlaskJsonifyCall() {
+        this = API::moduleImport("flask").getMember("jsonify").getACall()
+      }
+
+      override DataFlow::Node getBody() { result = this.getArg(0) }
```

Comment:
Looking at the documentation for `flask.jsonify`, it seems like it's really _any_ argument that's put into the response, so maybe this should be `getArg(_)` (or something similar)?

Reply:
good catch :+1: